test/libsolidity/syntaxTests/viewPureChecker/tload_not_pure.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
```

Comment:
@ekpyron, in this [comment](https://github.com/ethereum/solidity/pull/14737#discussion_r1444661571) you mentioned about checking `tload` and `tstore` behaviour on `pure` and `view`, which we already had on the initial work done by Hari. In that comment also, @cameel is asking for **Yul** tests... I think you were talking about Solidity tests like this and the following ones, right?

Reply:
Yeah, the pure checker tests are already here indeed, I just misread @cameel's comment, so you can disregard my comment there.